go.mod

```diff
@@ -51,7 +51,7 @@ require (
 	github.com/grafana/cuetsy v0.1.11 // @grafana/grafana-as-code
 	github.com/grafana/grafana-aws-sdk v0.25.0 // @grafana/aws-datasources
 	github.com/grafana/grafana-azure-sdk-go/v2 v2.0.1 // @grafana/partner-datasources
-	github.com/grafana/grafana-plugin-sdk-go v0.222.0 // @grafana/plugins-platform-backend
+	github.com/grafana/grafana-plugin-sdk-go v0.223.1-0.20240416132542-5d8bc8d25317 // @grafana/plugins-platform-backend
```

Comment:
TODO need to actually open a PR for this one https://github.com/grafana/grafana-plugin-sdk-go/compare/main...expose-serve-opts